requests leads to ImportError

> ..\.tox\dev\Lib\site-packages\_pytest\assertion\rewrite.py:168: in exec_module
    exec(co, module.__dict__)
test_highlevel.py:5: in <module>
    from ebdtable2graph import EbdTable
..\.tox\dev\Lib\site-packages\ebdtable2graph\__init__.py:5: in <module>
    from ebdtable2graph.plantuml import convert_graph_to_plantuml, convert_plantuml_to_svg_kroki
..\.tox\dev\Lib\site-packages\ebdtable2graph\plantuml.py:4: in <module>
    import requests  # pylint: disable=import-error
E   ModuleNotFoundError: No module named 'requests'


https://github.com/Hochfrequenz/ebd_docx_to_table/pull/12

--- a/setup.cfg
+++ b/setup.cfg
@@ -32,6 +32,7 @@ python_requires = >=3.11
 install_requires =
     attrs
     networkx
+    requests
     # write here line by line the dependencies for your package (from requirements.in)
 
 [options.packages.find]

